arch/inst/C/c.ldsp.yaml

```diff
@@ -6,26 +6,34 @@ name: c.ldsp
 long_name: Load doubleword from stack pointer
 description: |
   C.LDSP is an RV64C/RV128C-only instruction that loads a 64-bit value from memory
-  into register rd.
+  into register xd.
```

Comment:
With your new RV32 code merged in here, this statement is now incorrect, but we're probably both getting tired of this PR. ;-) And, the documentation will be updated in a later pass. If you feel inclined, change this to:
> For RV64, load a 64-bit value from memory into register xd. For RV32 with Zclsd extension, load a 64-bit value from memory into register pair xd, xd+1.